qiskit_nature/problems/second_quantization/electronic/electronic_structure_problem.py

```diff
@@ -124,6 +139,8 @@ def hopping_qeom_ops(
         """Generates the hopping operators and their commutativity information for the specified set
         of excitations.
 
+        You cannot call this method before calling `second_q_ops()`.
```

Comment:
I think it would be better if we turn it to 
```suggestion
        This method should can be used after calling `second_q_ops()`.
```